language/transaction-builder/generator/tests/generation.rs

```diff
@@ -298,6 +298,83 @@ fn test_that_java_code_compiles_and_demo_runs() {
     assert!(output.status.success());
 }
 
+#[test]
```

Comment:
I need to fix this file up a bit more. I forgot about it. The rest of the PR can be looked at though.

Reply:
Nice, this looks quite promising!

Reply:
Fixed up!